src/tito/release/aliases.py

```diff
@@ -0,0 +1,108 @@
+"""
+It is annoying to update `FedoraGitReleaser` branches every time a new Fedora
+is branched. We want branch aliases like `fedora-all` which evaluates to e.g.
+`f38 f39 rawhide`.
+
+The implementation is slightly over-engineered in comparison to what Tito needs
+because the goal is to move this in a separate, general-purpose python package.
+Then we can share the code with Packit and possibly others. See this packit
+code - https://github.com/packit/packit/blob/main/packit/config/aliases.py#L14
+"""
+
+import requests
+from tito.exception import TitoException
+
+
+def pdc_active_products():
+    """
+    Fetch the list of active products from PDC
+    https://pdc.fedoraproject.org/rest_api/v1/product-versions/
+    """
+    pdc = "https://pdc.fedoraproject.org"
+    url = "{0}/rest_api/v1/product-versions/?active=true".format(pdc)
```

Comment:
Is branched, but not released active? How we can differentiate the branched but not released one?

Reply:
I second this question.  From the "releaser" perspective, we want to ensure we publish also into the freshly branched fedora version (which I'm not sure is provided by PDC).